src/Umbraco.Core/PropertyEditors/PropertyValueConverterBase.cs

```diff
@@ -4,44 +4,57 @@
 namespace Umbraco.Core.PropertyEditors
 {
     /// <summary>
-    /// Provides a default overridable implementation for <see cref="IPropertyValueConverter"/> that does nothing.
+    /// Provides a default implementation for <see cref="IPropertyValueConverter" />.
     /// </summary>
+    /// <seealso cref="Umbraco.Core.PropertyEditors.IPropertyValueConverter" />
     public abstract class PropertyValueConverterBase : IPropertyValueConverter
     {
+        /// <inheritdoc />
         public virtual bool IsConverter(IPublishedPropertyType propertyType)
             => false;
 
+        /// <inheritdoc />
         public virtual bool? IsValue(object value, PropertyValueLevel level)
         {
             switch (level)
             {
                 case PropertyValueLevel.Source:
-                    return value != null && (!(value is string) || string.IsNullOrWhiteSpace((string) value) == false);
+                    // the default implementation uses the old magic null & string comparisons,
+                    // other implementations may be more clever, and/or test the final converted object values
+                    return value != null && (!(value is string stringValue) || !string.IsNullOrWhiteSpace(stringValue));
+                case PropertyValueLevel.Inter:
+                    return null;
+                case PropertyValueLevel.Object:
+                    return null;
                 default:
                     throw new NotSupportedException($"Invalid level: {level}.");
             }
         }
 
+        [Obsolete("This method is not part of the IPropertyValueConverter contract and therefore not used, use IsValue instead.")]
```

Comment:
Might be good to also mention that this will be removed in later versions since we'll prob just remove it in the netcore branch.

Reply:
Just by marking it obsolete [implicitly informs uses that the element will be removed in future versions of the product](https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute#remarks). But we can always change the message to make the more clear:

```suggestion
        [Obsolete("This method is not part of the IPropertyValueConverter contract, therefore not used and will be removed in future versions; use IsValue instead.")]
```